### PR TITLE
SAAS-309 textarea component

### PR DESCRIPTION
**Pull request by atymchuk**

### CODE REVIEW

File: packages/platform-core/src/components/Checkbox/CheckboxField.tsx [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { Field, FieldInputProps, FieldMetaState, UseFieldConfig } from 'react-fi
...
 import { cx } from 'emotion';
 import { getStyles } from './Checkbox.styles';
-import { Validator, compose } from '@/shared/validators';
+import { Validator, compose } from '../../shared/validators';
 
 export interface CheckboxProps extends UseFieldConfig<boolean> {
   disabled?: boolean;
   fieldClassName?: string;
-  label?: string;
+  label?: string | JSX.Element;
```
Comment (nicolalamacchia) [inline packages/platform-core/src/components/Checkbox/CheckboxField.tsx:11]: Why not using `ReactNode` here?

File: packages/platform-core/src/components/PasswordInput/PasswordInputField.tsx [hunk 2]
```diff
@@ -2,23 +2,23 @@ import React, { FC, useMemo } from 'react';
...
+ * Note: the validation error message will be displayed once the the input has been modified.
+ * To show the error message on blur you have to pass `showErrorOnBlur`.
  */
 export interface PasswordInputFieldProps extends UseFieldConfig<string> {
-  alwaysShowError?: boolean;
   className?: string;
   disabled?: boolean;
   fieldClassName?: string;
-  label?: string;
+  label?: string | JSX.Element;
```
Comment (nicolalamacchia) [inline packages/platform-core/src/components/PasswordInput/PasswordInputField.tsx:16]: Why not using `ReactNode` here?

File: packages/platform-core/src/components/TextInput/TextInputField.tsx [hunk 3]
```diff
@@ -2,23 +2,23 @@ import React, { FC, useMemo } from 'react';
...
+ * Note: the validation error message will be displayed once the the input has been modified.
+ * To show the error message on blur you have to pass `showErrorOnBlur`.
  */
 export interface TextInputFieldProps extends UseFieldConfig<string> {
-  alwaysShowError?: boolean;
   className?: string;
   disabled?: boolean;
   fieldClassName?: string;
-  label?: string;
+  label?: string | JSX.Element;
```
Comment (nicolalamacchia) [inline packages/platform-core/src/components/TextInput/TextInputField.tsx:16]: Why not using `ReactNode` here?

**Review by nicolalamacchia — DISMISSED** [review]

File: packages/platform-core/src/components/Checkbox/CheckboxField.tsx (hunk 1 above)
Comment (atymchuk) [reply packages/platform-core/src/components/Checkbox/CheckboxField.tsx]: Done

File: packages/platform-core/src/components/PasswordInput/PasswordInputField.tsx (hunk 2 above)
Comment (atymchuk) [reply packages/platform-core/src/components/PasswordInput/PasswordInputField.tsx]: Doen

File: packages/platform-core/src/components/TextInput/TextInputField.tsx (hunk 3 above)
Comment (atymchuk) [reply packages/platform-core/src/components/TextInput/TextInputField.tsx]: Done

**Review by nicolalamacchia — APPROVED** [review]

**Review by tiagomotasantos — COMMENTED** [review]
Tried it and everything LGTM, just good job 👍

**Pull request merged**